bin/docker_start

```diff
@@ -24,13 +24,16 @@ if ENV['AUTO_MIGRATE'] == '1'
       break
     rescue StandardError => e
       exception = e
-      sleep(1.second)
+      sleep(2.second)
     ensure
       connection.close unless connection.nil?
     end
   end
   raise exception unless connected
 
+  # the PG driver is slow to release its port binding: https://www.postgresql.org/message-id/30038.1494916166%40sss.pgh.pa.us
+  sleep(2.second)
+
   # create and migrate DB
   abort('Unable to migrate DB') unless system('bundle exec rails db:create db:migrate')
```

Comment:
Is this retry-able like the connection process above itself (if would that even be helpful)?

Reply:
Anything is retry-able if we want it to be. But in this case I don't think that would help. The previous steps should ensure the DB is up, and this should always work if the DB is up.